### PR TITLE
Commiting ASE to AKS migraion changes

### PR DESCRIPTION
**Pull request by vijayrajagopalan-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-1334

### Change description ###
ASE to AKS migration changes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: charts/rd-user-profile-api/values.demo.template.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+java:
+    # Don't modify below here
+  image: ${IMAGE_NAME}
```
Comment (KOTLAPRA23) [inline charts/rd-user-profile-api/values.demo.template.yaml:3]: This file is not required.

Comment (vijayrajagopalan-hmcts) [reply charts/rd-user-profile-api/values.demo.template.yaml]: Removed file as advised

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -23,27 +23,26 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
...
     env.IDAM_URL_TO_TEST = "https://idam-api.aat.platform.hmcts.net"
-    env.TEST_URL = "http://rd-user-profile-api-aat.service.core-compute-aat.internal"
 
     overrideVaultEnvironments(vaultOverrides)
     loadVaultSecrets(secrets)
 
     enableDbMigration('rd')
     installCharts()
+    
     // **This is for demo. Uncomment below for AAT/AKS deployments**
```
Comment (shreedhar-hmcts) [inline Jenkinsfile_CNP:43]: you can remove this comment since is not valid now since for env we require disableLegacyDeployment()

Comment (vijayrajagopalan-hmcts) [reply Jenkinsfile_CNP]: Removed comment.

**Pull request merged**